bff-service/app.js

```diff
@@ -0,0 +1,67 @@
+import express from 'express';
+import cors from 'cors';
+import axios from 'axios';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const app = express();
+const port = process.env.PORT || 4000;
+export const headers = {
+  'Access-Control-Allow-Origin': '*', // Required for CORS support to work
+  'Access-Control-Allow-Credentials': true, // Required for cookies, authorization headers with HTTPS
+};
+
+app.use(cors());
+app.use(express.json());
+
+let cache = {};
+
+setInterval(() => {
+  cache = {};
+}, 10000);
+
+app.all('*', async (req, res) => {
+  const url = req.originalUrl;
+  const serviceName = url.split('/')[1].toUpperCase();
+  const serviceURL = process.env[serviceName];
+  const cacheKey = req.method + '/' + serviceURL;
+
+  console.log('resource: ' + serviceName);
+  console.log('resource url: ' + serviceURL);
+  console.log('___');
+
+  if (serviceURL) {
+    if (cache[cacheKey]) {
```

Comment:
And if somebody did more than required, is it bad?

Reply:
Actually it's up to you, I don't insist on making some changes, because it's your pet project :)
But from my point of view developer should strictly follow the technical documentation and shouldn't do more than was required, even if it's easier to do :)